colearn/ml_interface.py

```diff
@@ -44,6 +70,13 @@ class ColearnModel(BaseModel):
     model: Optional[Any]
 
 
+def deser_model(model: Any) -> ColearnModel:
+    """
+    Helper function to recover a onnx model from its deserialized form
+    """
+    return onnx.load_model_from_string(model)
```

Comment:
It appears the return type here is wrong? It's returning a model, not a ColearnModel.

Reply:
Nice catch, fixed. Unfortunate that type safety isn't forced in python!